package.json

```diff
@@ -26,6 +26,7 @@
     "@types/mdx": "^2.0.11",
     "clsx": "^2.1.0",
     "cmdk": "^1.0.0",
+    "fast-glob": "^3.3.3",
```

Comment:
Quick question, what's this dependency for? 🤔

Reply:
It was missing from the package.json, we're using it to collect markdown files from our content directory.